code/_helpers/maths.dm

```diff
@@ -3,7 +3,8 @@
 #define ceil(x) (-round(-(x)))
 #define CEILING(x, y) ( -round(-(x) / (y)) * (y) )
 #define MULT_BY_RANDOM_COEF(VAR,LO,HI) VAR =  round((VAR * rand(LO * 100, HI * 100))/100, 0.1)
-#define PERCENT(val, max, places) round((val) / (max) * 100, !(places) || 10 ** -(places))
+/// The percentage of value in max, rounded to places: 1 = nearest 0.1 , 0 = nearest 1 , -1 = nearest 10, etc
+#define Percent(value, max, places) round((value) / (max) * 100, !(places) || 10 ** -(places))
```

Comment:
Why make this be in lowercase?

Reply:
Was looking through code and apparently, that's what they did on Bay.
Would be useful if we ever need to port something else from it.

Reply:
It's general styling that all macros be in all uppercase, but if it's for the sake of porting compatibility, I'll ask for @Cupax3's opinion.

Reply:
No need to port Bay's inconsistency too. Keep it UPPERCASE and include a comment that'll allow searches to detect the lowercase version so it's not as jarring to find as a full-on replacement, please.